UXWContributors.js

```diff
@@ -1,13 +1,104 @@
+const data = [ 
+  {
+    name: 'Keri',
+    role: 'Product Designer',
+    email: 'example@example.com',
+    image: '/assets/keri.jpg'
+  },
+  {
+    name: 'Laura',
+    role: 'Co-Organizer of UX Wizards',
+    email: 'example@example.com',
+    image: '/assets/laura.jpg'
+  },
+  {
+    name: 'Seiko',
+    role: 'Product Designer',
+    email: 'example@example.com',
+    image: '/assets/seiko.jpg'
+  },
+  {
+    name: 'Kaisen',
+    role: 'Software',
+    email: 'example@example.com',
+    image: '/assets/kaisen.jpg'
+  },
+  {
+    name: 'Javi',
+    role: 'Frontend Developer',
+    email: 'example@example.com',
+    image: '/assets/javi.jpg'
+  },
+  {
+    name: 'Max',
+    role: 'Product Designer',
+    email: 'example@example.com',
+    image: '/assets/max.jpg'
+  },
+  {
+    name: 'Sidney',
+    role: 'Product Designer',
+    email: 'example@example.com',
+    image: '/assets/sidney.jpg'
+  },
+  {
+    name: 'Dhruv',
+    role: 'Software',
+    email: 'example@example.com',
+    image: '/assets/dhruv.jpg'
+  },
+]
+
+const buttonStyle = {
+  outline: "0",
+  display: "inline-block",
+  padding: "8px",
+  "font-size":"15px",
+  color: "white",
+  "background-color": "#6331FD",
+  "text-align": "center",
+  cursor: "pointer",
+  width: "100%"
+}
+
+const titleStyle = {
+  color: "grey"
+}
+
+const columnStyle = {
+  float: "left",
+  width: "33.3%",
+  "margin-bottom": "16px",
+  padding: "0 8px"
+}
+
+const cardStyle = {
+  "box-shadow": "0 4px 8px 0 rgba(0, 0, 0, 0.2)",
+  "font-size": "10px"
+}
+
 function UXWContributors(props) {
   return (
     <div className="container">
-      <p>
-        Lucas ipsum dolor sit amet carnor needa chazrach rebo sebulba plagueis finis di veknoid ventress.
-        Maul lando vader sy hutt nomi. Gordin haruun bail jar. Fett mothma toydarian jerec greedo bothawui sesswenna artaru zuckuss.
-        Nassau cerea sern naboo. Tagge leia obi-wan thennqora antilles. Davin selkath jobal tavion metalorn organa muunilinst jettster.
-        jinn hutt ikrit vor. Bertroff kael hutt wharl oola dagobah bibble saleucami. Colton raioballo ansionian hapan billaba wampa organa.
-        Greedo argazdan jinn tof verpine bothan wedge.
-      </p>
+      <meta name="viewport" content="width=device-width, initial-scale=1" />
+      <h1>Meet The Ux Wizards!</h1>
+      <List>
+      {data.map(({name, role, email, image}) => (
+      <div className="row">
+        <div className="column" style={columnStyle}>
+          <div className="card" style={cardStyle}>

```

Comment:
**Suggestion**(non-blocking): I would consider making the whole card clickable if you're looking to add a contact button to it.